syncserver-common/Cargo.toml

```diff
@@ -4,6 +4,7 @@ version = "0.12.3"
 edition = "2021"
 
 [dependencies]
+actix-web = "3"
```

Comment:
It is unfortunate that actix web needs to be a dependency here. since only reason we pull it in is to use the `web::block` function to spawn tasks on blocking threads, which doesn't have anything to do with the syncstorage/tokenserver web API. Actix 4 exposes settings related to tokio's blocking threadpool, so once we upgrade to Actix 4, we'll be able to replace this dependency with tokio.